.travis.yml

```
@@ -27,6 +27,9 @@ env:
     - AWS_BOXI_EXPORT_ACCESS_KEY_ID=5678
     - AWS_BOXI_EXPORT_SECRET_ACCESS_KEY=5678
     - AWS_BOXI_EXPORT_BUCKET=5678
+    - AWS_LETTERS_EXPORT_BUCKET="dede"
+    - AWS_LETTERS_EXPORT_ACCESS_KEY_ID="dsa"
+    - AWS_LETTERS_EXPORT_SECRET_ACCESS_KEY="dsa"
```

```suggestion
    - AWS_LETTERS_EXPORT_ACCESS_KEY_ID=9012
    - AWS_LETTERS_EXPORT_SECRET_ACCESS_KEY=9012
    - AWS_LETTERS_EXPORT_BUCKET=9012
```

Just rearranging for consistency :)